### DISCUSSION TITLE
Kaiascan API

### DISCUSSION THREAD

**Discussion by PaulElisha** [kaiachain/kaia-dapp-mono#256] [Q&A]
I am exploring the free API from KaiaScan and noticed that it provides access to various blockchain data. However, I found that all the information can also be obtained directly from a node. In terms of functionality and ease of use, how does KaiaScan compare to Etherscan? Are there specific advantages or limitations of using KaiaScan's API over directly querying a node or using Etherscan?

**Comment by zxstim** [comment]
well there is no etherscan on kaia at the moment so this is one of your option. will send this feedback to the team for review.

**Reply by PaulElisha** [reply]
Alright, good.
